src/libDirectoryService/DirectoryService.cpp

```diff
@@ -86,7 +86,14 @@ void DirectoryService::StartSynchronization() {
     }
   };
 
+  auto func2 = [this]() -> void {
+    bool result = m_mediator.m_lookup->FetchDSInfoLoop();
+    if (!result) {
```

Comment:
We don't need a temporary variable "result", suggest to just use if(!m_mediator.m_lookup->FetchDSInfoLoop())

Reply:
ok